tests/scripts/run-xunit-perf.cmd

```diff
@@ -377,9 +384,6 @@ rem   Sends text to the console screen, no matter what (even when the script's
 rem   output is redirected). This can be useful to provide information on where
 rem   the script is executing.
 rem ****************************************************************************
-  if defined _debug (
```

Comment:
Could we delete the entire `:print_to_console` function? I don't think anybody uses it anymore, right?